test cases/frameworks/4 qt/meson.build

```diff
@@ -55,11 +55,15 @@ foreach qt : ['qt4', 'qt5', 'qt6']
       method : get_option('method')
     )
     # XML files that need to be compiled with the uic tol.
-    prep += qtmodule.compile_ui(sources : 'mainWindow.ui', method: get_option('method'))
+    prep += qtmodule.compile_ui(
```

Comment:
Is it possible to extend the test to also use compile_moc?

Reply:
TBH, I'm not familiar enough with Qt to know now to do a simple test for that. My usecase was for .ui files. I just extended it to moc as well, but in our codebase, it seems the current `moc_compile` is working without modifications.